azure-quantum/azure/quantum/workspace.py

```diff
@@ -257,6 +257,27 @@ def list_jobs(
 
         return result
 
+    def get_targets(self) -> Dict[str, List[str]]:
+        """Returns a dictionary of provider IDs and lists of targets
+        that are available.
+
+        :return: Targets, keyed by provider IDs
+        :rtype: Dict[str, List[str]]
+        """
+        return {
+            provider.id: [
+                target.id for target in provider.targets
+            ] for provider in self._client.providers.get_status()
+        }
+
+    def get_job_quotas(self) -> List[Dict[str, Any]]:
+        """Get a list of job quotas for the given workspace.
+
+        :return: Job quotas
+        :rtype: List[Dict[str, Any]]
+        """
+        return [q.as_dict() for q in self._client.quotas.list()]

```

Comment:
`self._client.quotas` should be wrapped in its own `_get_quotas_client`

Reply:
Sure, good idea to keep it consist